src/pdfboxing/info.clj

```diff
@@ -14,7 +14,7 @@
 (defn- getter
   "private helper function to be used by about-doc"
   [obj method-str]
-  (let [get-method (str "get" (string/capitalize method-str))]
+  (let [get-method (str "get" (string/join (map string/capitalize (string/split method-str #"-"))))]
```

Comment:
This is good, but what do you think about maybe indenting this line like this:

```clojure
  (let [get-method (str "get"
                        (string/join
                         (map string/capitalize
                              (string/split method-str #"-"))))]
```

This is just to keep the line length under 72 characters.